nano/node/bootstrap.hpp

```diff
@@ -289,6 +289,13 @@ class bootstrap_listener final
 std::unique_ptr<seq_con_info_component> collect_seq_con_info (bootstrap_listener & bootstrap_listener, const std::string & name);
 
 class message;
+enum class bootstrap_server_type
```

Comment:
Maybe a separate PR, but probably wanna rename boostrap_server to tcp_server at this point (and move to separate file?), and bootstrap_server_type - > tcp_server_type.

Reply:
Good idea! But probably better separate PR with just renames